cmd/troubleshoot/cli/run.go

```diff
@@ -91,23 +95,47 @@ func runTroubleshoot(v *viper.Viper, arg []string) error {
 			mainBundle = supportbundle.ConcatSpec(mainBundle, supportBundle)
 		}
 
-		for i, additionalDoc := range multidocs {
-			if i == 0 {
-				continue
-			}
-			additionalDoc, err := docrewrite.ConvertToV1Beta2([]byte(additionalDoc))
-			if err != nil {
-				return errors.Wrap(err, "failed to convert to v1beta2")
-			}
-			obj, _, err := decode(additionalDoc, nil, nil)
-			if err != nil {
-				return errors.Wrapf(err, "failed to parse additional doc %d", i)
-			}
-			multidocRedactors, ok := obj.(*troubleshootv1beta2.Redactor)
-			if !ok {
-				continue
+		parsedRedactors, err := supportbundle.ParseRedactorsFromSpec(multidocs)
+		if err != nil {
+			return errors.Wrap(err, "failed to parse redactors from doc")
+		}
+		additionalRedactors.Spec.Redactors = append(additionalRedactors.Spec.Redactors, parsedRedactors...)
+	}
+
+	if v.GetBool("load-specs-from-secrets") {
```

Comment:
While clear, this is a fairly long flag for a user to type. Does it/can it also have a short version? If it can't have a long and short versions can we reduce the flag to be shorter?